CHANGELOG.md

```diff
@@ -21,6 +21,7 @@ reason `Cloud usernames must be less than or equal to`.
 - Added new exception `Py42UserNotOnListError` to throw error if `remove()` throws `404` when the user is not on a detection list.
 
 - Added new exception `Py42TooManyRequestsError` to raise errors with 429 HTTP status code.
+- Added new method `sdk.securitydata.get_all_file_events()` to retrieve all events when events are more than 10,000
```

Comment:
Should have a newline between this entry and the one above

Reply:
Added.